src/components/remote_control/src/resource_allocation_manager_impl.cc

```diff
@@ -200,5 +216,27 @@ void ResourceAllocationManagerImpl::OnDriverDisallowed(
   list_of_rejected_resources.push_back(module_type);
 }
 
-ResourceAllocationManager::~ResourceAllocationManager() {}
+void ResourceAllocationManagerImpl::RefreshAllocatedResourcesState() {
+  LOG4CXX_AUTO_TRACE(logger_);
```

Comment:
@dev-gh accepted. Now added notification AM->PluginManager->RC plugin->ResourceAllocationManager when application was unregistered so there is no more need to do related cleanup manually in all places. Please see https://github.com/LuxoftAKutsan/sdl_core/pull/199/commits/a4d880829583b394f673410fb577272cc1ebf3fa